From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Sun, 11 Dec 2016 18:00:13 +0100
Subject: firmware-utils: tplink-safeloader: update support list for CPE210/510/...

Adds support for new EU and US variants, removes a few strings that were
never actually used and have been removed from the stock firmwares.

Signed-off-by: Matthias Schiffer <mschiffer@universe-factory.net>

diff --git a/tools/firmware-utils/src/tplink-safeloader.c b/tools/firmware-utils/src/tplink-safeloader.c
index 4938f74..4a38dff 100644
--- a/tools/firmware-utils/src/tplink-safeloader.c
+++ b/tools/firmware-utils/src/tplink-safeloader.c
@@ -169,8 +169,13 @@ static const char cpe210_support_list[] =
 	"SupportList:\r\n"
 	"CPE210(TP-LINK|UN|N300-2):1.0\r\n"
 	"CPE210(TP-LINK|UN|N300-2):1.1\r\n"
-	"CPE220(TP-LINK|UN|N300-2):1.0\r\n"
-	"CPE220(TP-LINK|UN|N300-2):1.1\r\n";
+	"CPE210(TP-LINK|US|N300-2):1.1\r\n"
+	"CPE210(TP-LINK|EU|N300-2):1.1\r\n"
+	"CPE220(TP-LINK|UN|N300-2):1.1\r\n"
+	"CPE220(TP-LINK|US|N300-2):1.1\r\n"
+	"CPE220(TP-LINK|EU|N300-2):1.1\r\n"
+	;
+
 /**
    The support list for CPE210/220/510/520
 */
@@ -178,8 +183,13 @@ static const char cpe510_support_list[] =
 	"SupportList:\r\n"
 	"CPE510(TP-LINK|UN|N300-5):1.0\r\n"
 	"CPE510(TP-LINK|UN|N300-5):1.1\r\n"
-	"CPE520(TP-LINK|UN|N300-5):1.0\r\n"
-	"CPE520(TP-LINK|UN|N300-5):1.1\r\n";
+	"CPE510(TP-LINK|UN|N300-5):1.1\r\n"
+	"CPE510(TP-LINK|US|N300-5):1.1\r\n"
+	"CPE510(TP-LINK|EU|N300-5):1.1\r\n"
+	"CPE520(TP-LINK|UN|N300-5):1.1\r\n"
+	"CPE520(TP-LINK|US|N300-5):1.1\r\n"
+	"CPE520(TP-LINK|EU|N300-5):1.1\r\n"
+	;
 
 /**
    The support list for C2600
